src/xprof.app.src

```diff
@@ -1,13 +1,17 @@
 {application, xprof,
  [{description, "Visual BEAM profiler"}
- ,{vsn, "1.1.0"}
+ ,{vsn, "1.2.0"}
  ,{registered, []}
  ,{mod, {'xprof_app', []}}
  ,{applications,
    [kernel
    ,stdlib
    ,cowboy
    ,lager
+   ,jsone
+   ,hdr_histogram
+   ,edown
```

Comment:
Not entirely true, because building release with `distillery` complains that those apps are not included for _hdr_histogram_ (it is a warning).